From e14ff784f3aedb13bd5d103fba3186d4cd5ba00a Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Sun, 05 Jun 2022 16:00:05 +0300
Subject: [PATCH] Revert "[SW-77251] gaudi3: update sram base address"

Revert rom recovery implementation for secured firmware update via
inband and OOB.

This reverts commit c1bebe6d944ddf0accbc44ae31c18c6ccce42c72.

Change-Id: Idd098f46529eacf13d32d5bca46a1a3e404101a2
---

diff --git a/arch/gaudi3/arc/soc.h b/arch/gaudi3/arc/soc.h
index e304f57..98e3061 100644
--- a/arch/gaudi3/arc/soc.h
+++ b/arch/gaudi3/arc/soc.h
@@ -42,8 +42,6 @@
 #define SZ_64K				0x00010000
 #define SZ_128K				0x00020000
 #define SZ_256K				0x00040000
-#define SZ_16M				0x01000000
-#define SZ_32M				0x02000000
 #define SZ_256MB			0x10000000
 /* 2MB for Gaudi2 . TODO: Confirm for gaudi3 */
 #define FLASH_VALID_MASK		0x1FFFFF
@@ -88,9 +86,6 @@
 #define HL_FWSRAM_OFFSET		0x3140000
 #define HL_PCIE_FWSRAM_OFFSET		0x7F00000
 #define SP_SRAM_BASE			(HL_GEN_MEM_BASE + HL_SCRATCHPAD_SRAM_OFFSET)
-#define HL_SRAM_MODE0_SIZE		(SZ_32M + SZ_16M) /* single die mode */
-#define HL_SRAM_MODE0_BASE		(HL_FW_MEM_BASE - HL_SRAM_MODE0_SIZE)
-#define SDRAM_BASE			HL_SRAM_MODE0_BASE
 
 #define HL_FW_MEM_BASE_FULL		0x0300007FD0000000ull
 #define HL_GEN_MEM_BASE_FULL		HL_FW_MEM_BASE_FULL
diff --git a/common/memory.h b/common/memory.h
index 00f69b9..29d28ae 100644
--- a/common/memory.h
+++ b/common/memory.h
@@ -7,7 +7,9 @@
 #define _MEMORY_H_
 
 #include <stdint.h>
-#include <soc.h>
+
+/* TODO: re-verify below offsets */
+#define SDRAM_BASE		0x10000000
 
 #define MEM_GP_BASE		(SDRAM_BASE + 0x40000)	/* SRAM_BASE + 256K */
 #define MEM_GP_SZ		0xC0000			/* 768K */
